source/tutorials/metadata.rst

```diff
@@ -0,0 +1,138 @@
+Metadata in QIIME 2
+===================
+
+Metadata enables users of QIIME 2 to supplement their analyses with information relevant to their study. This manifests itself in many ways: indicating the barcode sequence when demultiplexing sequences, sorting and grouping taxonomic assignments in the taxonomic bar plots, or exploring diversity metrics with visualizations like ``alpha-group-significance`` or ``beta-group-significance``. In QIIME 1, metadata (also known as the metadata mapping file), was a user-defined file that contained these study-specific fields. QIIME 2 expands upon this idea, allowing users to provide their own study metadata (via a TSV file), or, by viewing other study-specific artifacts as metadata. Examples of both are presented in the following.
+
+Begin by creating a directory to conduct the work in:
+
+.. command-block::
+   :no-exec:
+
+   mkdir qiime2-metadata-tutorial
+   cd qiime2-metadata-tutorial
+
+Metadata from a mapping file
+----------------------------
+
+Metadata is typically defined in a mapping file, similar to the QIIME 1 mapping file. The QIIME 2 development team hasn't adopted a standard set of criteria for the metadata mapping file, but at present the following minimum requirements are enforced:
+
+- The file must be a tab-separated text file
+- The first line of the file is the column labels, tab-separated
+- The header values in the first line must be unique (i.e. no duplicate values)
+- There is at least one line of data following the header line
+- The first column is the "identifier" column (either Sample ID or Feature ID, depending on the axis)
+- All of the values in the first column must be unique (i.e. no duplicate values)
+
+Metadata mapping files can be validated using Keemei_. Generally speaking, metadata mapping files `that work in QIIME 1`_ should work in QIIME 2.
+
+To get started with understanding metadata mapping files, first download a sample mapping file:
+
+.. TODO: Update this link
+.. download::
+   :url: https://data.qiime2.org/2017.5/tutorials/moving-pictures/sample_metadata.tsv
+   :saveas: sample-metadata.tsv
+
+This file can be opened and edited in a text editor, Google Sheets, or Keemei_. There is also a ``metadata tabulate`` visualization in QIIME 2 that allows for interactive sorting and filtering of metadata:
+
+.. command-block::
+   qiime metadata tabulate \
+     --m-input-file sample-metadata.tsv
+     --o-visualization tabulated-sample-md.qzv
+
+.. question::
+   Based on the table in ``tabulated-sample-md.qzv``, how many samples are associated with Subject 1? How many samples are associated with the ``gut`` body site? Hint: use the search box and/or the column sorting options to assist with this query.
+
+Metadata from a QIIME 2 artifact
+--------------------------------
+
+As eluded to above, QIIME 2 also supports the notion of viewing artifacts as metadata (where it makes sense). An example of this is alpha-diversity output artifacts.
```

Comment:
Suggestion: *As eluded to above, QIIME 2 also supports the notion of viewing artifacts as metadata (where it makes sense).* -> *As eluded to above, QIIME 2 also supports the notion of viewing some artifacts as metadata.*